### ISSUE TITLE
Spider victra is broken

### ISSUE THREAD

**Issue by scraperbot** [alltheplaces/alltheplaces#3284]
During the global build at 2021-10-20-14-42-48, spider **victra** failed with **0 features** and **0 errors**.

Here's [the log](https://data.alltheplaces.xyz/runs/2021-10-20-14-42-48/logs/victra.txt) and [the output](https://data.alltheplaces.xyz/runs/2021-10-20-14-42-48/output/victra.geojson) ([on a map](https://data.alltheplaces.xyz/map.html?show=https://data.alltheplaces.xyz/runs/2021-10-20-14-42-48/output/victra.geojson))

**Comment by CloCkWeRX** [comment]
https://victra.com/wp-admin/admin-ajax.php?action=store_search&lat=40.75369&lng=-73.99916&max_results=25&search_radius=50 wordpress now.

**Comment by CloCkWeRX** [comment]
```
import scrapy

from locations.storefinders.wp_store_locator import WPStoreLocatorSpider
from locations.hours import DAYS_EN


class VictraSpider(WPStoreLocatorSpider):
    name = "victra"
    item_attributes = {"brand": "Victra"}
    allowed_domains = [
        "victra.com",
    ]
    days = DAYS_EN
    time_format = "%I:%M %p"

```

**Comment by CloCkWeRX** [comment]
Looks like it will need point searching.

**Comment by CloCkWeRX** [comment]
https://victra.com/sitemap_index.xml
https://victra.com/wpsl_stores-sitemap.xml
https://victra.com/wpsl_stores-sitemap2.xml

**Comment by davidhicks** [comment]
Fixed by #8232
